src/fsharp/FSComp.txt

```diff
@@ -716,7 +716,7 @@ tcUnnamedArgumentsDoNotFormPrefix,"The unnamed arguments do not form a prefix of
 871,tcConstructorsIllegalForThisType,"Constructors cannot be defined for this type"
 872,tcRecursiveBindingsWithMembersMustBeDirectAugmentation,"Recursive bindings that include member specifications can only occur as a direct augmentation of a type"
 873,tcOnlySimplePatternsInLetRec,"Only simple variable patterns can be bound in 'let rec' constructs"
-874,tcOnlyRecordFieldsAndSimpleLetCanBeMutable,"Only record fields and simple, non-recursive 'let' bindings may be marked mutable"
+874,tcOnlyRecordFieldsAndSimpleLetCanBeMutable,"Mutable 'let' bindings can neither be recursive nor defined in recursive module"
```

Comment:
Could we try using a bit simpler English? I can imagine how some non-native speakers could be confused by "neither/nor" here, actually making the error text more difficult to grasp. 🙂 
Maybe something like "Mutable 'let' bindings can't be recursive or defined in recursive modules"?

Reply:
Should probably be "recursive modules or namespaces". A namespace can be rec too.

Reply:
Should note that @cartermp agreed earlier on this text proposal: https://github.com/dotnet/fsharp/issues/3642#issuecomment-665104096

Reply:
I'd like the adjusted text. 'tis what reviews are good for :)